### PR TITLE
Enable Galaxy publish for c.general

### PR DESCRIPTION
**Pull request by gundalow**
Branch protections have been enabled for `community.general`.

I'm not exactly sure which `templates` to add. We are using Shippable for CI at the moment.

Do we need any Zuul configuration files adding to https://github.com/ansible-collections/community.general ?

Once this has been seen to work I'll add it for `community.*` in a follow-up PR

### CODE REVIEW

**Comment by gundalow** [comment]
What do I need to do to fix

> The project "github.com/ansible-collections/community.general" was not
> found.  All projects referenced within a Zuul configuration must first
> be added to the main configuration file by the Zuul administrator.

File: zuul.d/projects.yaml [hunk 1]
```diff
@@ -75,6 +75,12 @@
       - integrated-queue
       - publish-to-galaxy
 
+- project:
```
Comment (pabelanger) [inline zuul.d/projects.yaml:78]: This needs to be a 2nd PR. We first need to load the project into zuul, then we can assign jobs.

File: zuul/tenants.yaml [hunk 2]
```diff
@@ -27,6 +27,7 @@
           - ansible-collections/cisco.ios
           - ansible-collections/cisco.iosxr
           - ansible-collections/cisco.nxos
+          - ansible-collections/community.general
```
Comment (pabelanger) [inline zuul/tenants.yaml:30]: Do you want zuul to do the merge or still have humans manually do it.  If humans, this needs to move the line 95 below

**Comment by gundalow** [comment]
recheck

File: zuul.d/projects.yaml [hunk 3]
```diff
@@ -75,6 +75,12 @@
...
+    name: github.com/ansible-collections/community.general
+    templates:
+      - system-required
```
Comment (pabelanger) [inline zuul.d/projects.yaml:81]: so, this isn't needed. We only use it when we gate projects.

File: zuul.d/projects.yaml [hunk 4]
```diff
@@ -75,6 +75,12 @@
...
+      - publish-to-galaxy
```
Comment (pabelanger) [inline zuul.d/projects.yaml:82]: This won't work either, as it adds gate jobs. For now, we'll manually add jobs.

Comment (pabelanger) [reply zuul.d/projects.yaml:82]:
```
    third-party-check:
      jobs:
        - build-ansible-collection
    pre-release:
      jobs:
        - release-ansible-collection-galaxy
    release:
      jobs:
        - release-ansible-collection-galaxy
```

**Pull request merged**